### PR TITLE
[OPT-553] to feature/OPT-551

### PR DESCRIPTION
**Pull request by pablosantamartae**
[OPT-553] Update public documentation

[OPT-553]: https://iriusrisk.atlassian.net/browse/OPT-553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md [hunk 1]
```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
...
-## Threats
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM threats automatically
-## Mitigations
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM mitigations automatically
\ No newline at end of file
+
+## Threats and Mitigations
+No need to be in the mapping file.
+
+StartLeft will extract OTM hreats and threat instances, as well as OTM mitigations
```
Comment (abausac) [inline docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:120]: "StartLeft will extract OTM **threats** and ..."

Comment (pablosantamartae) [reply docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md]: Done!

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md [hunk 2]
```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
...
-## Mitigations
-No need to be in the mapping file. StartLeft will detect them and will fill the OTM mitigations automatically
\ No newline at end of file
+
+## Threats and Mitigations
+No need to be in the mapping file.
+
+StartLeft will extract OTM threats and threat instances, as well as OTM mitigations
+and mitigation instances from the same MTMT threat. Moreover the OTM component to which the OTM threat instance will be added
+is the destination component from the MTMT threat.
```
Comment (abausac) [inline docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:122]: "is the destination component **`of`** the MTMT threat."

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md [hunk 3]
```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
...
+
+Currently, Startleft maps threats and mitigations from two kind of templates:
```
Comment (abausac) [inline docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:124]: "Currently, Startleft maps threats and mitigations from two **`kinds`** of templates:"

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md [hunk 4]
```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
...
+    }
+  ]
+}
+```
+
+There is a special case when the MTMT threat has no ". Consider" pattern in the "Description" field. In such case, no OTM
+mitigation nor OTM mitigation instance will be mapped, only the OTM threat and threat instance can be mapped.
+
+### Azure Threat Model Template (AzureArchitecture) template
+#### A MTMT threat without "Steps" field
```
Comment (abausac) [inline docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:178]: "#### A MTMT threat without **`a "Steps"`** field"

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md [hunk 5]
```diff
@@ -113,7 +113,142 @@ the mapping file. Let's see how to do it:
...
+
+```json
+{
+  "name": "Enforce communication between clients and Azure Postgres DB to be over SSL/TLS by enabling the Enforce SSL connection feature on the server",
+  "id": "1",
+  "riskReduction": 100,
+  "description": "Enforce communication between clients and Azure Postgres DB to be over SSL/TLS by enabling the Enforce SSL connection feature on the server. Check that the connection strings used to connect to MySQL databases have the right configuration (e.g. ssl = true or sslmode=require or sslmode=true are set). Refer: <a href=\"https://aka.ms/tmt-th154a\">https://aka.ms/tmt-th154a</a> Configure MySQL server to use a verifiable SSL certificate (needed for SSL/TLS communication). Refer: <a href=\"https://aka.ms/tmt-th154b\">https://aka.ms/tmt-th154b</a>"
+}
+```
+#### A MTMT threat with "Steps" field
```
Comment (abausac) [inline docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:208]: "#### A MTMT threat with **`a "Steps"`** field"

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md (hunk 2 above)
Comment (pablosantamartae) [reply docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:122]: Done!

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md (hunk 3 above)
Comment (pablosantamartae) [reply docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:124]: Done!

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md (hunk 4 above)
Comment (pablosantamartae) [reply docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:178]: Done!

File: docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md (hunk 5 above)
Comment (pablosantamartae) [reply docs/startleft-processors/external-threat-model/mtmt/MTMT-Mapping.md:208]: Done!

**Review by abausac — APPROVED** [review]

**Pull request merged**